web/app/themes/dp-timber-theme/index.php

```diff
@@ -16,7 +16,7 @@
 $context          = Timber::context();
 $context['posts'] = Timber::get_posts();
 $context['foo'] = 'Shayaan';
-$templates        = array( 'index.twig' );
+$templates        = array( 'advance.twig' );
```

Comment:
add a conditional way to provide the new twig suggestion.

Reply:
I will go with the partials & Include method as we have done it in basic syntax